amd_openvx/openvx/ago/ago_haf_cpu_geometric.cpp

```diff
@@ -2434,6 +2450,28 @@ vx_uint32     srcImageStrideInBytes
 			_mm_store_si128(dstNext++, pixels2);
 			_mm_store_si128(dstNext++, pixels1);
 		}
+
+		if (fallbackToC)
+		{
+			unsigned char *src = (unsigned char *) pSrcImage;
+			unsigned char *dst = (unsigned char *) pchDst;
+			unsigned char *dstNext = (unsigned char *) (pchDst + dstImageStrideInBytes);
+			unsigned char *dstLast = dst + dstWidth - 1;
+
+			while (dst < dstLast)
+			{
+				*dst = *src;
+				*dst++;
+				*dst = *src;
+				*dst++;
+
+				*dstNext = *src;
+				*dstNext++;
```

Comment:
@latinovic What does *dstNext++ means? Did you mean incrementing dst address. If so please remove 2468 and 2469 and replace with *dstNext++ = *src.

Reply:
@milkoleporis can you look at this comment? I can merge this branch.

Reply:
Yes, *dstNext++ is incrementing the address.
I've changed it to *dstNext++ = *src in the https://github.com/kiritigowda/MIVisionX/pull/1